server/plugin/api.go

```diff
@@ -1289,23 +1296,26 @@ func (p *Plugin) createIssue(w http.ResponseWriter, r *http.Request, userID stri
 		return
 	}
 
-	rootID := issue.PostID
-	if post.RootId != "" {
-		rootID = post.RootId
+	postID := ""
+	channelID := issue.ChannelID
+	message := fmt.Sprintf("Created GitHub issue [#%v](%v)", result.GetNumber(), result.GetHTMLURL())
+	if post != nil {
+		postID = post.Id
+		channelID = post.ChannelId
+		message += fmt.Sprintf(" from a [message](%s)", permalink)
 	}
 
-	message := fmt.Sprintf("Created GitHub issue [#%v](%v) from a [message](%s)", result.GetNumber(), result.GetHTMLURL(), permalink)
 	reply := &model.Post{
 		Message:   message,
-		ChannelId: post.ChannelId,
-		RootId:    rootID,
-		ParentId:  rootID,
+		ChannelId: channelID,
+		RootId:    postID,
```

Comment:
A bit of a nit pick: If a user triggers the slash command from the RHS, the response post in put in the center channel. It's not a big deal and can be solved later.